crates/node/src/shell/testing/node.rs

```diff
@@ -380,6 +380,21 @@ impl MockNode {
             .0
     }
 
+    pub fn next_masp_epoch(&mut self) -> Epoch {
```

Comment:
is the mut ref needed?
```suggestion
    pub fn next_masp_epoch(&self) -> Epoch {
```

Reply:
Yes cause internally it calls `next_epoch` which requires a mutable ref